<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
    <widget class="QWidget" name="centralwidget">
     <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>DocumentScannerAI</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="font">
             <font>
                <pointsize>18</pointsize>
                <weight>75</weight>
                <bold>true</bold>
             </font>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectButton">
            <property name="text">
             <string>Select PDF Document</string>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fileLabel">
            <property name="text">
             <string>No file selected.</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="analyzeButton">
            <property name="text">
             <string>Analyze Document</string>
            </property>
            <property name="enabled">
             <bool>false</bool>
            </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="resultsTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Analysis results will appear here...</string>
            </property>
         </widget>
        </item>
     </layout>
    </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
